game/templates/game/board.html

```
@@ -35,29 +46,25 @@
-    </svg>
-  </button>
-  <button class="m-fab-btn m-fab-resign" title="Resign"
-    onclick="document.getElementById('resignBtn').click()">
-    <svg width="16" height="16" viewBox="0 0 24 24" fill="currentColor">
-      <path d="M4 3h14l-4 4.5 4 4.5H4V3z"/>
-      <line x1="4" y1="21" x2="4" y2="3" stroke="currentColor" stroke-width="2" stroke-linecap="round"/>
-    </svg>
-  </button>
-</div>
+    <div class="m-fab" id="mFab">
+        <button class="m-fab-btn" title="Pause" onclick="document.getElementById('pauseBtn').click()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)